db/migrate/20180406045858_add_unique_index_to_lists_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexToListsUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_index(:lists_users, [:list_id, :user_id], unique: true)
```

Comment:
I remembered something else. When a column-combination index like this is added, because of the way it's built, it can replace any index for the first column mentioned (`list_id`). What you'll need to do is:

1. `rails db:rollback`
1. edit this migration to add the line `remove_index :lists_users, :list_id`
1. `rails db:migrate`
1. save the changes as a new commit, I'll can help clean this up to amend an earlier commit

Reply:
Actioned in 2ab39262c890748035cc1743d5c5e5584db613a7